### PR TITLE
new file: es.po

### PR DESCRIPTION
**Pull request by mjtullio**
New translation file in Spanish.

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: prefs.js:76
+msgid "Behaviour"
+msgstr "Comportamiento"
+
+#: prefs.js:85
+msgid "Disable activities hot corner"
+msgstr "Desactivar menú de actividades en la esquina de la pantalla"
```
Comment (LinxGem33) [inline po/es.po]: is there a shorter translation for this ?

something like = Deshabilitar actividades hot corner

**Comment by LinxGem33** [comment]
appreciate the translation is there a way you can shorten the translations as some are a bit to long , i have given a example above.

kind regards

**Comment by mjtullio** [comment]
I will try to make them as short as possible.

**Comment by LinxGem33** [comment]
thank you.

**Comment by mjtullio** [comment]
As regards your comments, phrases are shorter now. I've checked them and I think they fit nicely in the menu window. Cheers.

**Pull request merged**